packages/core/src/audio/AudioSource.ts

```
@@ -0,0 +1,256 @@
+
+  /**
+   * The audio clip to play.
+   */
+  get clip(): AudioClip {
+    return this._clip;
+  }
+
+  set clip(value: AudioClip) {
+    const lastClip = this._clip;
+    if (lastClip !== value) {
+      lastClip && lastClip._addReferCount(-1);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)